srcpkgs/libxcb/template

```diff
@@ -1,20 +1,20 @@
 # Template file for 'libxcb'
 pkgname=libxcb
-version=1.14
-revision=2
+version=1.15
+revision=1
 build_style=gnu-configure
 configure_args="--disable-build-docs --enable-xinput --enable-xkb"
 hostmakedepends="libtool pkg-config xorg-util-macros xcb-proto"
 makedepends="xcb-proto libXdmcp-devel libXau-devel"
```

Comment:
xcb-proto can be removed from the -devel subpackage's $depends as someone pointed out on IRC

Reply:
Cross seem to depend on it:

`configure: error: Package requirements (xcb-proto >= 1.14) were not met:`

```
pkg:libxcb host:x86_64 target:x86_64 cross:n result:OK
pkg:libxcb host:x86_64 target:aarch64 cross:y result:FAILED
```

Reply:
Add it only on cross then?

Reply:
@motorto I get that it needs it for the build, but I don't see why it should be in -devel package's dependencies.

Reply:
You mean removing the $(makedepends) from the depends,right ?

Reply:
`$makedepends` is a variable that expands to `xcb-proto libXdmcp-devel libXau-devel`, -devel package's dependencies should be just `libXdmcp-devel libXau-devel` because xcb-proto is not required there. So yes, remove `$makedepends` from the -devel subpackage's `$depends`, but also add the required dependencies that were previously put there with `$makedepends`.

Reply:
added, thanks for the heads up. And sorry for don`t understanding it at first